pandas/tests/extension/test_categorical.py

```diff
@@ -87,6 +87,31 @@ def test_memory_usage(self, data):
         # Is this deliberate?
         super().test_memory_usage(data)
 
+    def test_contains(self, data, data_missing):
+        # GH-37867
+        # na value handling in Categorical.__contains__ is deprecated.
+        # See base.BaseInterFaceTests.test_contains for more details.
```

Comment:
This duplicates what you have in `tests/arrays/categorical/test_operators.py ` ?

Reply:
Unfortunately not: `Categorical` already had a `__contains__` method and it's more permissive than the new one. So, here we have below `assert na_value_type in data_missing`, while the base tests method is `assert na_value_type not in data_missing` (notice the `not`).

na values is also more complicated in categoricals, because in some cases we want to accept `pd.NaT` and in other cases not. I'd like to take it in another round (or let it slide)

Reply:
This was a mistake, I will remove it.

Reply:
To be clear, I was not referring to the base tests that this is overriding, but the original test you added to `tests/arrays/categorical/test_operators.py ` which also tests this more permissive behaviour?

Reply:
Yeah. I'll delete the ones in `tests/arrays/categorical/test_operators.py`.